### ISSUE TITLE
Images on websites

### ISSUE THREAD

**Issue by Darth-Ness** [Darth-Ness/Katana-HTML-Renderer#9]
When ever I travel to a website that uses images, instead of rendering the correct image; It renders the image not found file.
I think this has something to do with the findTarget function in renderMain.py. More specifically, this line in paticalura 
```python
 i = item.find("src=")
```
Can any one see if you can find a web page with working images.

**Comment by DelightedDuck** [comment]
Hmm thats strange, i know for sure my site was working too. It seems that sites don't load at all :frowning_face:.

**Comment by Darth-Ness** [comment]
> Hmm thats strange, i know for sure my site was working too. It seems that sites don't load at all frowning_face.

Yeah, I did a test and it does work on local webpages. Yep, I'm not surprised there.

**Comment by Darth-Ness** [comment]
There's a chance this will be fixed in version 2.
